packages/front-end/components/Admin/EditOrganization.tsx

```diff
@@ -57,7 +61,19 @@ const EditOrganization: FC<{
           minLength={3}
           onChange={(e) => setName(e.target.value)}
         />
-        {showExternalId && (
+        {isCloud() && (
+          <div className="mt-3">
+            License Key
+            <input
+              type="text"
+              className="form-control"
+              value={licenseKey}
+              minLength={3}
```

Comment:
Does this `minLength` make it so you can't edit an organization's name if they don't have a license key?  Maybe we should move this logic to the submit handler.  If licenseKey is not empty, then validate the length/format.

Reply:
No, since it is not required it doesn't complain about being blank.  I left it in thinking it can't hurt, but it also not needed at all.  We already have validation in the submit handler to see if the license is valid license, and an "Invalid license key" is a more accurate error message than "This field needs to be at least three characters you have entered 1", so I'll just remove it.